src/Decorator/MessageBusDecorator.php

```diff
@@ -34,15 +28,14 @@ public function dispatch($message, array $stamps = []): Envelope
         if ($jobMessage instanceof JobMessageInterface) {
             try {
                 $this->scheduleMessage($jobMessage);
-            } catch (WriteTypeIntendException $e) {
-                null;
+            } catch (WriteTypeIntendException) {
```

Comment:
should we do something in a catch?

Reply:
I am not 100% sure about the logic here, because I can't even find the exception in the called functions. But I didn't want to risk anything and just kept the logic while removing the unnecessary line.

Reply:
Fair maybe this exception ain't even thrown anymore :P